dev-util/babeltrace/babeltrace-2.0.6.ebuild

```diff
@@ -38,16 +39,19 @@ BDEPEND="${RDEPEND}
 "
 
 src_configure() {
+	filter-lto #925900
```

Comment:
https://bugs.gentoo.org/925900#c2

Reply:
The real question IMO is why it's building with -Werror.

We don't want to care about maybe-uninitialized as an LTO blocking concern.

Reply:
Hence the actual bug here is https://bugs.gentoo.org/920929